packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, DOMProps, FocusableRefValue, StyleProps} from '@react-types/shared';
+import {classNames, createFocusableRef, SlotProvider, useStyleProps} from '@react-spectrum/utils';
+import {DropZoneProps, FileTriggerContext, Provider, DropZone as RACDropZone} from 'react-aria-components';
+import {mergeProps} from '@react-aria/utils';
+import React, {ReactNode, Ref, useImperativeHandle, useRef} from 'react';
+import styles from '@adobe/spectrum-css-temp/components/dropzone/vars.css';
+export interface SpectrumDropZoneProps extends DropZoneProps, DOMProps, StyleProps, AriaLabelingProps {
+  children: ReactNode,
+  isFilled?: boolean, 
+  bannerMessage?: string
```

Comment:
More of a naming question for the team to discuss, but I think something like `replaceMessage` or `replaceLabel` might be better here.

Reply:
I prefer `replaceMessage` over `replaceLabel` because it might sound like we're adding a label for accessibility which isn't the case. And with `replaceMessage`, I like how it is clear what the prop does but like you said, up for opinions.